respx/models.py

```diff
@@ -226,7 +226,7 @@ def __init__(
             self.pass_through = None
             self._match_func = method
         else:
-            self.method = method
+            self.method = method.upper()
```

Comment:
Is this a decent approach? 
We could also make a top level constant `respx.GET == "GET" 
and encourage folk to use that instead of the string literals?

Reply:
I think this is the correct place and way to solve the issue.

About the constants, I'd say we skip them to align with `HTTPX`. If HTTPX adds constants for the http methods in the future, then those can be uses with RESPX.